src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-//#define PARSING_TESTS_DUMP
+#define PARSING_TESTS_DUMP
```

Comment:
Is there a reason this was disabled before?

Reply:
This should be reverted. Only turn on to troubleshoot locally I think.

---
In reply to: [324927208](https://github.com/dotnet/roslyn/pull/38713#discussion_r324927208) [](ancestors = 324927208)